### PR TITLE
Documentation and restructuring of component transport tests

### PR DESCRIPTION
**Pull request by TomFischer**

### CODE REVIEW

File: web/content/docs/benchmarks/hydro-component/goswami.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+## Overview
+
+This benchmark features an Henry-like variable-density flow and transport setup that evaluates saltwater intrusion on laboratory scale. The original benchmark was published by Goswami & Clement (2007); OpenGeoSys5 has been verified for all steady and transient states of the original benchmark (see Kolditz et al., 2012).
+
+For the setup and parameterization, see the chapter "Density dependent flow - The Goswami Problem" in Kolditz et al. (2012).
+
+
+## Problem description
+
+The Goswami-Clement benchmark is based on experiment observations for intruding and receding saltwater in a laboratory-scale sand tank. Here, we compare numerical results of ogs6 to those of OGS5 and the original observation data.
```
Comment (endJunction) [inline web/content/docs/benchmarks/hydro-component/goswami.md]: nit-picking: Sometimes ogs6, sometimes OGS5; both are wrong: OGS-6, OGS-5.

File: web/content/docs/benchmarks/hydro-component/hydro-component.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+{{< img src="../gif/DiffusionAndStorage.gif" title="*Diffusion and Storage*">}}
+
+#### Diffusion, Storage, and Advection
+
+Left side boundary conditions for this setup are pressure $p=1$ and concentration $c=1$.
+
+[The *Diffusion, Storage, and Advection* project file is here. ](../../../../../Tests/Data/Parabolic/ComponentTransport/SimpleSynthetics/DiffusionAndStorageAndAdvection.prj)
+
+{{< img src="../gif/DiffusionAndStorageAndAdvection.gif" title="*Diffusion, Storage, and Advection*">}}
```
Comment (endJunction) [inline web/content/docs/benchmarks/hydro-component/hydro-component.md]: I'd either skip the title (as it is already in the section name) or provide more details on what one is seeing there.
Looks a little strange when rendered: [web benchmarks](https://jenkins.opengeosys.org/job/ufz/job/ogs/job/PR-1941/11/Web/)

**Comment by endJunction** [comment]
web/content/docs/benchmarks/hydro-component/elder.md:31: trailing whitespace.

**Comment by bilke** [comment]
@TomFischer @waltherm Please configure `git lfs` as [described here](https://docs.opengeosys.org/docs/devguide/getting-started/prerequisites)!!! I fixed it with a6746b6 for now.
